templates/customernotification.html

```diff
@@ -10,41 +10,68 @@
         This notification is for your OpenShift managed cluster running OpenShift Data Foundation.
         <br><br>
         <strong>
+
+        {{ $FirstTimeErrorCephClusterCriticallyFull := true }}
+        {{ $FirstTimeErrorCephOSDNearFull := true }}
+        {{ $FirstTimeErrorCephOSDCriticallyFull := true }}
+        {{ $FirstTimeErrorPersistentVolumeUsageNearFull := true }}
+        {{ $FirstTimeErrorPersistentVolumeUsageCritical := true }}
+        {{ $FirstTimeErrorCephClusterReadOnly := true }}
+        {{ $FirstTimeErrorCephClusterNearFull := true }}
+
         {{ range .Alerts.Firing }}
             {{ range .Labels.SortedPairs }}
                 {{ if eq .Name "alertname" }}
-                        {{ if eq .Value "CephClusterCriticallyFull" }}
+                        {{ if and (eq .Value "CephClusterCriticallyFull" ) (eq $FirstTimeErrorCephClusterCriticallyFull true ) }}
```

Comment:
@akshayashu Isn't this problematic? will it not skip any future updates on the same alert type?

Reply:
@nb-ohad please don't merge this MR, I think I'll have to make more changes.

Reply:
/hold